### ISSUE TITLE
Incorrect identify system prompt while generate SDG

### ISSUE THREAD

**Issue by alfandindarahmawan** [instructlab/sdg#406]
Hi team,
While i running SDG used  mixtral-8x7b as the teacher and the model for student i used granint-3.0-2b-instruct. the system prompt on sdg result like this .
```
I am a Red Hat® Instruct Model, an AI language model developed by Red Hat and IBM Research based on the granite-3.0-8b-base model. My primary role is to serve as a chat assistant.
```
Which is incorrect because i used granint-3.0-2b-instruct but the data result form sdg set the system prompt is `granite-3.0-8b-base `.

**Comment by mairin** [comment]
@alfandindarahmawan also found the likely place in the code causing this:

https://github.com/instructlab/instructlab/blob/main/src%2Finstructlab%2Fcommon.py

**Comment by cdoern** [comment]
I think, generally, that sys prompt is for any granite 3.0 architecture model. 

If there is a specific one per-model in the granite 3 family, this won't scale well in adding support for new models, but @jaideepr97 can comment more here.

**Comment by jaideepr97** [comment]
> I think, generally, that sys prompt is for any granite 3.0 architecture model.
> 
> If there is a specific one per-model in the granite 3 family, this won't scale well in adding support for new models, but @jaideepr97 can comment more here.

what @cdoern said
currently we only have 1 sys prompt for all granite based models

**Comment by Maxusmusti** [comment]
Yes, this is what we wanted to do for for earlier versions, but had run into some initial push back. This will definitely be revisited soon.
